src/Throttle/Entity/CacheCount.php

```diff
@@ -79,7 +78,7 @@ protected function toArray(): array
     {
         return [
             'count' => $this->count,
-            'ttl' => $this->ttl,
+            'ttl'   => $this->ttl,
         ];
     }
 }
```

Comment:
May I ask which set of codestyle guidelines you use for this project? So I could adhere to them as well :)

Reply:
Sorry, i think my PHPStorm settings got reset to default :)